server/network.go

```diff
@@ -377,6 +380,14 @@ func (lc *localNetwork) updateSubnetInfo(ctx context.Context) error {
 		}
 	}
 
+	for subnetID, nodeNames := range lc.nw.GetSubnetParticipants() {
```

Comment:
I would not extende ANR local for this, but use pchain gesubnets and getcurrentvalidators to get the subnet participants. this is also more suitable in order to rehuse network saved on snapshots. will work without problem in that case. as a general rule: when possible, just try to get information from pchain (from the blockchain), so things like snapshots does not need to save more information that needed